### PR TITLE
feat: Improve section on string interpolation in webhook payloads

### PR DESCRIPTION
**Pull request by valekjo**
This PR improves the section on string interpolation, as discussed on slack.

### CODE REVIEW

File: sources/platform/integrations/webhooks/actions.md [hunk 1]
```diff
@@ -68,6 +66,34 @@ The syntax of a variable therefore is: `{{oneOfAvailableVariables}}`. The variab
 }
 ```
 
+#### String interpolation
+
+The payload template **is not** a valid JSON by default. The resulting payload is, but not the template. In some cases this is limiting, so there is also updated syntax available, that allows to use templates that provide the same functionality, and are valid JSON at the same time.
```
Comment (TC-MO) [inline sources/platform/integrations/webhooks/actions.md:71]:
```suggestion
The payload template _is not_ a valid JSON by default. The resulting payload is, but not the template. In some cases this is limiting, so there is also updated syntax available, that allows to use templates that provide the same functionality, and are valid JSON at the same time.
```

File: sources/platform/integrations/webhooks/actions.md [hunk 2]
```diff
@@ -68,6 +66,34 @@ The syntax of a variable therefore is: `{{oneOfAvailableVariables}}`. The variab
...
+
+If the string being interpolated only contains the variable, the actual variable value is used in the payload. For example `"{{eventData}}"` results in an object. If the string contains more than just the variable, the string value of the variable will occur in the payload:
+
+```json
+{ "text": "My user id is {{userId}}" }
+{ "text": "My user id is abf6vtB2nvQZ4nJzo" }
+```
+
+
+To turn on this new syntax, use "Interpolate variables in string fields" switch. In API it's called `shouldInterpolateStrings`. The field is always `true` when integrating Actors or tasks.
```
Comment (TC-MO) [inline sources/platform/integrations/webhooks/actions.md:95]:
```suggestion
To turn on this new syntax, use **Interpolate variables in string fields** switch within Apify Console. In API it's called `shouldInterpolateStrings`. The field is always `true` when integrating Actors or tasks.
```

File: sources/platform/integrations/webhooks/actions.md (hunk 1 above)
Comment (TC-MO) [reply sources/platform/integrations/webhooks/actions.md:71]: Emphasis should be done with italics bold should be usually used for noting UI elements when describing something.

File: sources/platform/integrations/webhooks/actions.md (hunk 2 above)
Comment (TC-MO) [reply sources/platform/integrations/webhooks/actions.md:95]: I am correct to assume that this refers to this switch within Console? 
image

**Review by mvolfik — APPROVED** [review]
Love it, thanks a lot! Maybe worth noting that in A2A only the second syntax is available? Or is that already explained in the docs page?

**Comment by TC-MO** [comment]
I see that Vale is failing, fix for that issue is on another PR, waiting to be approved & merged

**Comment by valekjo** [comment]
> Love it, thanks a lot! Maybe worth noting that in A2A only the second syntax is available? Or is that already explained in the docs page specific for A2A?

@mvolfik 

It's noted there as "The field is always `true` when integrating Actors or tasks.". Do you have another idea how to phrase it?

**Review by TC-MO — APPROVED** [review]

**Comment by mvolfik** [comment]
Ah, missed that part. Maybe we could link to this section from https://docs.apify.com/platform/integrations/actors, but either way, LGTM

**Pull request merged**